docs/data/date-pickers/time-range-picker/time-range-picker.md

```diff
@@ -6,7 +6,7 @@ packageName: '@mui/x-date-pickers-pro'
 materialDesign: https://m2.material.io/components/date-pickers
 ---
 
-# Time Range Picker [<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan)🚧
+# Time Range Picker <a title="Pro plan" href="/x/introduction/licensing/#pro-plan"><span class="plan-pro"></span></a>🚧
```

Comment:
Have you considered using the [MD syntax ](https://www.markdownguide.org/basic-syntax/#adding-titles)for adding a `title`? 🤔 
Or was there problems parsing it?
```suggestion
# Time Range Picker [<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan "Pro plan")
```

Reply:
really, Oh wow, I never heard of this. It doesn't work in our docs because we didn't wire the title 😬 https://github.com/mui/material-ui/blob/master/packages/markdown/parseMarkdown.js#L336. But we could make it work. 

Who has a strong preference? We need to update core as well https://master--material-ui.netlify.app/experiments/docs/headers/#header-with-pro-plan